src/platform/datapath_raw_socket.c

```diff
@@ -43,6 +43,57 @@ CxPlatGetSocket(
     return Socket;
 }
 
+BOOLEAN
+CxPlatIsRouteReady(
+    _In_ QUIC_CONNECTION *Connection,
+    _In_ CXPLAT_ROUTE_RESOLUTION_CALLBACK_HANDLER Callback,
+    _In_ BOOLEAN PathChallenge
+) {
+    QUIC_PATH* Path = &Connection->Paths[0];
```

Comment:
This breaks the abstraction boundaries. Platform can't use, or know about, any QUIC layer structures.

Reply:
then how core code distinguish underlying datapath is raw or normal?